src/test/java/org/folio/circulation/resources/ReorderQueueTestDataSource.java

```diff
@@ -0,0 +1,38 @@
+package org.folio.circulation.resources;
+
+import java.util.Arrays;
+
+public class ReorderQueueTestDataSource {
+
+  @SuppressWarnings("unused")
+  public static Object[][] provideDataForReorderQueueTwiceTest() {
```

Comment:
What do the examples with position 50 in them mean?

My understanding is the request positions should be contiguous, so these situations should be disallowed?

Reply:
Cate responded that positions have to be sequential, so I've added additional validation for this and adjusted unit tests.